Lib/ast.py

```diff
@@ -368,6 +368,69 @@ def walk(node):
         yield node
 
 
+def compare(
+    a,
+    b,
+    /,
+    *,
+    compare_types=True,
+    compare_fields=True,
+    compare_attributes=False,
+):
+    """
+    Recursively compare given 2 AST nodes. If *compare_types* is False, the
+    field values don't be checked whater they belong to same type or not. If
```

Comment:
Typo - I think "whater" should be "whether". In above doc file also

Reply:
Yes, thanks you.